pnpm-workspace.yaml

```diff
@@ -8,3 +8,18 @@ onlyBuiltDependencies:
   - msw
   - node-pty
   - protobufjs
+# Without this, electron-builder will pull in electron-builder-squirrel-windows (which we don't need)
+# and that dep will in turn pull electron-winstaller (which we also don't need). pnpm will then
+# start complaining that electron-winstaller is not in onlyBuiltDependencies.
+#
+# We could instead solve this by adding electron-winstaller to ignoredBuiltDependencies, but why
+# pull in deps that we don't use? Ultimately, the problem is that electron-builder-squirrel-windows
+# is defined as a peer dep of one of electron-builder deps, while in reality it's more like an
+# optional dep.
+autoInstallPeers: false
```

Comment:
npm does that by default since v7, pnpm started doing it too at some point.

I'm reading the RFD and just shaking my head the whole time. https://github.com/npm/rfcs/blob/main/implemented/0025-install-peer-deps.md

app-builder-lib has electron-builder-squirrel-windows as a peer dep, but from what I can tell it's more like an optional dep. Even [the docs](https://www.electron.build/squirrel-windows.html) tell you to install that squirrel-windows package _if_ you want to use Squirrel.Windows.

In any case, if I run `pnpm -i --resolution-only`, I see no complaints about peer deps. If I remove the squirrel dep from `ignoreMissing`, pnpm shows an error about a missing peer dep, which indicates to me that it's the only problematic peer dep in our project.